base_classes/NXprocess.nxdl.xml

```diff
@@ -45,12 +50,35 @@
     </field>
     <group type="NXnote">
         <doc>
-            The note will contain information about how the data was processed
-            or anything about the data provenance. 
-            The contents of the note can be anything that the processing code 
-            can understand, or simple text.
+            The note will contain information about how the data was
+            processed or anything about the data provenance. The
```

Comment:
Indeed that is exactly how we are using it also, could mention to folks using NXnote here for provenance.